langcodes/__init__.py

```diff
@@ -964,195 +939,155 @@ def standardize_tag(tag: str, macro: bool=False) -> str:
-    98
-
-    >>> # European Portuguese is a bit different from the Brazilian most common dialect
-    >>> tag_match_score('pt', 'pt-PT')
+    >>> tag_match_score('zh-HK', 'zh-MO')   # Chinese is similar in Hong Kong and Macao
+    97
+    >>> tag_match_score('en-AU', 'en-GB')   # Australian English is similar to British English
     96
-    >>> # UK and US English are also a bit different
-    >>> tag_match_score('en-GB', 'en-US')
+    >>> tag_match_score('en-IN', 'en-GB')   # Indian English is also similar to British English
     96
-    >>> # Swiss German speakers will understand standardized German
-    >>> tag_match_score('gsw', 'de')
+    >>> tag_match_score('es-PR', 'es-419')  # Peruvian Spanish is Latin American Spanish
     96
-    >>> # Most German speakers will think Swiss German is a foreign language
-    >>> tag_match_score('de', 'gsw')
-    0
+    >>> tag_match_score('en-US', 'en-GB')   # American and British English are somewhat different
+    94
+    >>> tag_match_score('es-MX', 'es-ES')   # Mexican Spanish is different from Spanish Spanish
+    92
+    >>> # Serbian has two scripts, and people might prefer one but understand both
+    >>> tag_match_score('sr-Latn', 'sr-Cyrl')
+    95
+    >>> # European Portuguese is different from the most common form (Brazilian Portuguese)
+    >>> tag_match_score('pt', 'pt-PT')
+    92
+    >>> # Norwegian Bokmål and Danish are basically regional varieties of the same language
+    92
 
-    A match strength of 90 represents languages with a considerable amount of
-    overlap and some amount of mutual intelligibility. People will probably be
-    able to handle the difference with a bit of discomfort.
+    A score of 86 to 90 indicates that people who use the desired language
+    are demographically likely to understand the supported language, even if
+    the languages themselves are unrelated. There are many languages that have
+    a one-way connection of this kind to English or French.
 
-    Algorithms may have more trouble, but you could probably train your NLP on
-    _both_ languages without any problems. Below this match strength, though,
-    don't expect algorithms to be compatible.
+    >>> tag_match_score('ta', 'en')  # Tamil to English
+    86
+    >>> tag_match_score('mg', 'fr')  # Malagasy to French
+    86
 
-    >>> tag_match_score('no', 'da')  # Norwegian Bokmål is like Danish
-    90
-    >>> tag_match_score('id', 'ms')  # Indonesian is like Malay
-    90
-    >>> # Serbian language users will usually understand Serbian in its other script.
-    >>> tag_match_score('sr-Latn', 'sr-Cyrl')
+    This is also used for highly related languages with a noticeable distinction,
+    such as Indonesian and Malay, or the two written forms of Norwegian.
+
+    >>> tag_match_score('nn', 'nb')  # Nynorsk to Norwegian Bokmål
     90
+    >>> tag_match_score('ms', 'id')  # Malay to Indonesian
+    86
+    >>> tag_match_score('af', 'nl')  # Afrikaans to Dutch
+    86
```

Comment:
The peculiarities of this matching function makes me just want to give up on Unicode's exact scale, like the golang version does. But I'll reword the part that introduces those examples.